deploy/cognito/userpool.tf

```diff
@@ -0,0 +1,38 @@
+variable env {}
+
+resource "aws_cognito_user_pool" "users" {
+  name = "hallebarde-${var.env}"
+
+  admin_create_user_config {
+    allow_admin_create_user_only = true
+  }
+
+  mfa_configuration = "OFF"
+
+  schema {
+    attribute_data_type = "String"
+    name = "email"
+    required = true
+    mutable = false
+
+    string_attribute_constraints {
+      max_length = 100
+      min_length = 1
+    }
+  }
+
+  schema {
+    attribute_data_type = "String"
+    name = "phone"
+    required = false
+    mutable = false
+
+    string_attribute_constraints {
```

Comment:
Why did you put these values ? Just curious.

Reply:
That is legacy from a previous test. I will delete the phone part (I am not 100% clear about keeping the email if we request the email oauth scope).